reana_client/utils.py

```diff
@@ -156,6 +158,13 @@ def _prepare_kwargs(reana_yaml):
             )
             validate_environment(reana_yaml, pull=pull_environment_image)
 
+        workspace = reana_yaml.get("workspace", {}).get("root_path")
+        if workspace:
+            display_message(
+                "Verifying workspace in REANA specification file...", msg_type="info",
+            )
+            _validate_workspace(reana_yaml["workspace"]["root_path"], access_token)
```

Comment:
Very minor: probably `workspace` (as defined in line 161) can be used here instead of `reana_yaml["workspace"]["root_path"]`